### DISCUSSION TITLE
Zepp OS v3.0 API Development Roadmap

### DISCUSSION THREAD

**Discussion by MyoungXUE** [zepp-health/zeppos-forum#59] [Announcements]
Hi everyone, back on January 18, we held an online developer event on the topic of announcing the Zepp OS 3.0 Roadmap and related updates.

See https://www.eventbrite.com/e/zepp-os-30-apis-product-roadmap-technical-updates-tickets-510416398187 for details.

I'm sharing the presentation here so that you developers can easily learn about Zepp OS v3.0.

[Zepp OS v3.0 API Development Updates.pdf](https://github.com/zepp-health/zeppos-forum/files/10531840/Zepp.OS.v3.0.API.Development.Updates.pdf)

**Comment by vaas1993** [comment]
When will the official version be released? Which models of devices will be supported?

**Reply by RinnetenseiQ** [reply]
Yea, I also want to know

**Reply by mimimix** [reply]
Up

**Reply by juanitadion** [reply]
Hi @vaas1993, @RinnetenseiQ and @Mimimix,

Excellent Question, and thank you for expressing your curiosity. 

We always follow the company's official policy - October 2023 for Zepp OS v3.0.  The devices will be officially announced at the same time.

**Comment by T-Bone82** [comment]
on which OS version will you offer the 3.0 update? i have a gtr4 with zepp os 2.0. should i get the 3.0 update when  it ready?

**Comment by Cotrox** [comment]
Are there any info about the 3.0 release?
